apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -50,6 +51,50 @@ export const ANDROID_CRL_URL =
 export const ANDROID_PLAY_INTEGRITY_URL =
   "https://www.googleapis.com/auth/playintegrity";
 
+export const MailConfig = t.type({
+  mailFeatureFlag: t.boolean,
+  mailSender: t.intersection([t.string, NonEmptyString]),
+  mailupSecret: t.intersection([t.string, NonEmptyString]),
+  mailupUsername: t.intersection([t.string, NonEmptyString]),
+});
+
+export type MailConfig = t.TypeOf<typeof MailConfig>;
+
+export const WALLET_ACTIVATION_EMAIL_SUBJECT =
+  "IT Wallet - Aggiungi i tuoi documenti al Portafoglio di IO";
+export const WALLET_ACTIVATION_EMAIL_TEXT =
+  "IT Wallet - Aggiungi i tuoi documenti al Portafoglio di IO";
+export const WALLET_ACTIVATION_EMAIL_FAQ_LINK =
+  "https://io.italia.it/documenti-su-io/faq/#n1_12";
+export const WALLET_ACTIVATION_EMAIL_HANDLE_ACCESS_LINK =
+  "https://ioapp.it/it/accedi/?refresh=true";
+
+export const getMailConfigFromEnvironment: RE.ReaderEither<
+  NodeJS.ProcessEnv,
+  Error,
+  MailConfig
+> = pipe(
+  sequenceS(RE.Apply)({
+    mailFeatureFlag: pipe(
+      readFromEnvironment("MailFeatureFlag"),
+      RE.map(booleanFromString),
+      RE.orElse(() => RE.right(false)),
+    ),
+    mailSender: pipe(
+      readFromEnvironment("MailSender"),
+      RE.chainEitherKW(parse(NonEmptyString, "Invalid mail sender")),
```

Comment:
if the environment variable is not present here you are giving the default value "Invalid mail sender". I don't think that's what you want

Reply:
Are you sure? This line allows the func to throw an error if I don't set any config variables called `MailSender` (same for `MailupSecret` and `MailupUsername`).

For example, if I start the func without the `MailupUsername` env variable, the func throws an error similar to:

`Error: Worker was unable to load entry point "dist/main.js": unable to find "MailupUsername" in node environment`